spec/components/works/available_date_component_spec.rb

```diff
@@ -14,11 +14,17 @@
     work_form.prepopulate!
   end
 
-  it 'checks the immediate release radio button' do
+  it 'checks the immediate release radio button by default' do
     expect(rendered.css('#release_immediate[@checked]')).to be_present
     expect(rendered.css('#release_embargo[@checked]')).not_to be_present
   end
 
+  it 'requires the date fields to be filled in' do
```

Comment:
quibble:   requires _all_ the date fields to be filed in ?

Reply:
Sounds good. Change made.